### PR TITLE
[cmd] Broken contract migration

### PR DESCRIPTION
**Pull request by ramtinms**
This PR adds a migration which fixes the broken contracts.

### CODE REVIEW

File: cmd/util/ledger/migrations/broken_contract_migration.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		}
+	} else {
+		// if not exist log and return gracefully
+		fmt.Println("warning account does not exist: ", addressInHex)
+	}
+	return nil
+}
+
+func GetKeyManagerContractContent() []byte {
+	return []byte(`import KeyManager from 0x840b99b76051d886
```
Comment (ramtinms) [inline cmd/util/ledger/migrations/broken_contract_migration.go:50]: @psiemens could you please double-check the account address that keyManager is loaded from.

File: fvm/transactionFeeDeductor.go [hunk 2]
```diff
@@ -39,9 +41,10 @@ func (d *TransactionFeeDeductor) deductFees(
 	proc *TransactionProcedure,
 	sth *state.StateHolder,
 	programs *programs.Programs,
+	span opentracing.Span,
 ) (errors.Error, error) {
-
 	feeTx := deductTransactionFeeTransaction(proc.Transaction.Payer, ctx.Chain.ServiceAddress())
+	feeTx.SetTraceSpan(span)
```
Comment (ramtinms) [inline fvm/transactionFeeDeductor.go:47]: added this to trace fee deduction steps as well.

File: cmd/util/ledger/migrations/broken_contract_migration.go [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// BrokenContractMigration fixes some of the early contracts that has been broken due to cadence upgrades.
```
Comment (psiemens) [inline cmd/util/ledger/migrations/broken_contract_migration.go:11]:
```suggestion
// BrokenContractMigration fixes some of the early contracts that have been broken due to Cadence upgrades.
```

File: cmd/util/ledger/migrations/storage_v4.go [hunk 4]
```diff
@@ -242,6 +242,16 @@ var tokenForwardingLocationMainnet = common.AddressLocation{
...
+var tokenHolderKeyManagerLocationMainnet1 = common.AddressLocation{
+	Address: common.BytesToAddress([]byte{0x93, 0x7c, 0xbd, 0xee, 0x13, 0x5c, 0x65, 0x6c}),
+	Name:    "TokenHolderKeyManager",
+}
+
+var tokenHolderKeyManagerLocationMainnet2 = common.AddressLocation{
+	Address: common.BytesToAddress([]byte{0x7f, 0x56, 0x0d, 0x7e, 0x3f, 0xf0, 0x4a, 0x31}),
+	Name:    "TokenHolderKeyManager",
+}
+
```
Comment (ramtinms) [inline cmd/util/ledger/migrations/storage_v4.go:254]: Hey @psiemens could you please double-check these as well.

File: cmd/util/ledger/migrations/broken_contract_migration.go (hunk 1 above)
Comment (psiemens) [reply cmd/util/ledger/migrations/broken_contract_migration.go:50]: Yep, this is correct. 👍

File: cmd/util/ledger/migrations/storage_v4.go (hunk 4 above)
Comment (psiemens) [reply cmd/util/ledger/migrations/storage_v4.go:254]: These are correct 👍

**Review by psiemens — APPROVED** [review]

**Review by Kay-Zee — APPROVED** [review]

**Pull request merged**